timetracker/entries/templates/entries.html

```diff
@@ -61,7 +61,7 @@
       <h2>Entries</h2>
       <ul>
         {% for entry in entry_list %}
-        <li>{{ entry.description }}</li>
+        <li><font color=green><b>{{ entry.start.date }}</b></font>: ({{ entry.project }}) {{ entry.description }}<br><br> </li>
```

Comment:
The `font` tag doesn't exist in modern HTML :)

<b style="color: green"> is acceptable... but CSS is better :)